inc/views/base/modcp/modlogs.twig

```diff
@@ -0,0 +1,90 @@
+{% extends 'layouts/master.twig' %}
+
+{% block head %}
+    <title>{{ mybb.settings.bbname }} - {{ lang.modlogs }}</title>
+{% endblock head %}
+
+{% block body %}
+    <table width="100%" border="0" align="center">
+        <tr>
+            {# To do: modcp_nav #}
+            <td valign="top">
+                <table border="0" cellspacing="{{ theme.borderwidth }}" cellpadding="{{ theme.tablespace }}" class="tborder">
+                    <tr>
+                        <td class="thead" align="center" colspan="5"><strong>{{ lang.modlogs }}</strong></td>
+                    </tr>
+                    <tr>
+                        <td class="tcat"><span class="smalltext"><strong>{{ lang.username }}</strong></span></td>
+                        <td class="tcat" align="center"><span class="smalltext"><strong>{{ lang.date }}</strong></span></td>
+                        <td class="tcat" align="center"><span class="smalltext"><strong>{{ lang.action }}</strong></span></td>
+                        <td class="tcat" align="center"><span class="smalltext"><strong>{{ lang.information }}</strong></span></td>
+                        <td class="tcat" align="center"><span class="smalltext"><strong>{{ lang.ip }}</strong></span></td>
+                    </tr>
+                    {% for logitem in modlogs %}
+                        {% include 'modcp/modlogs_logitem.twig' %}
```

Comment:
As I said in the warning logs item, it might make sense to move the row template into its own folder, `modcp/modlogs/item.twig`. We should also make sure to stick to a common naming scheme, so either call them all `row` or `item`. I've been calling them `row`: https://github.com/mybb/mybb/blob/develop/1.9/inc/views/base/usercp/subscribed_threads/row.twig